### DISCUSSION TITLE
How can I deploy webpage from WGLMakie?

### DISCUSSION THREAD

**Discussion by yuehhua** [JuliaPlots/Makie.jl#4210] [Q&A]
I have create an interactive web app from WGLMakie + JSServe.
I want to deploy it as a standalone html (maybe + javascript) webpage so that I can deploy it.
How can I do that? I cannot find any tutorials on the document.

**Comment by tlienart** [comment]
(removing my comment as I missed the "interactive" in your message and so my response is not useful to your case)

looking forward to Simon's demo indicated in his message below!

**Comment by SimonDanisch** [comment]
Can you give some more context? E.g. do you need to have a connection to a running Julia session, or do you want to export a static page? 
I wanted to prepare a demo for this soon...

**Comment by yuehhua** [comment]
I just found there are some examples in JSServe.jl. [Here](https://github.com/SimonDanisch/JSServe.jl/blob/master/examples/basic_offline_makie.jl) is for a standalone static page, and [another](https://github.com/SimonDanisch/JSServe.jl/blob/master/examples/serve.jl) serves a Julia session.
Both of them requires to change some widgets into JSServe's widgets. It's a bit hacky.
For my current project, I am developing a physical animation. There are some sliders for users to control some parameters.
Some 2D and 3D plots are going to refresh on the screen simultaneously. It can be a standalone web page or running a Julia session.
For future, a Julia session backend would require to calculate some values and read/write to files or database.
